<launch>

  <!-- args corresponding to environment variables -->
  <arg name="UAV_NAME" default="$(env UAV_NAME)" />

    <!-- other args -->
  <arg name="standalone" default="true" />

  <arg unless="$(arg standalone)" name="nodelet" value="load" />
  <arg     if="$(arg standalone)" name="nodelet" value="standalone" />
  <arg unless="$(arg standalone)" name="nodelet_manager" value="$(arg UAV_NAME)_vision_nodelet_manager" />
  <arg     if="$(arg standalone)" name="nodelet_manager" value="" />

  <group ns="$(arg UAV_NAME)">

    <node name="tomasuv_tf_publisher" pkg="nodelet" type="nodelet" args="$(arg nodelet) mrs_general/TomasuvTfPublisher $(arg nodelet_manager)" output="screen">

      <rosparam file="$(find mrs_general)/config/tomasuv_tf_publisher.yaml" />

      <param name="uav_name" value="$(arg UAV_NAME)" />

        <!-- Subscribers -->
      <remap from="~odom_main_in" to="odometry/odom_main" />
      <remap from="~odom_optflow_in" to="odometry/odom_optflow" />
      <remap from="~odom_brick_in" to="odometry/odom_brick" />
      <remap from="~odom_stable_in" to="odometry/odom_stable" />

    </node>

  </group>

</launch>
